wishlist.html

```diff
@@ -0,0 +1,152 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+      <meta charset="UTF-8">
+      <meta http-equiv="X-UA-Compatible" content="IE=edge">
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+      <title>Wishlist</title>
+      <script src="https://kit.fontawesome.com/4b28f70eeb.js" crossorigin="anonymous"></script>
+      <link rel="stylesheet" href="./css/wishlist.css">
+</head>
+<body>
+
+      <header>
+            <nav class="navbar bg-dark">
+                  <div class="nav-title">
+                        <h1 class="text-xl"><a class="btn">BookStore</a></h1>
```

Comment:
You should link the pages using the anchor tags or you can remove that tag if you don't want to redirect to anywhere.

And BookStore is supposed to be the brand so the class of logo might be suitable here.

Reply:
ok, thank you for the review and I will make a change.